test/e2e/run_test.go

```diff
@@ -48,6 +48,28 @@ var _ = Describe("Podman run", func() {
 		Expect(session).Should(Exit(0))
 	})
 
+	It("podman generates different names for successive containers", func() {
+		SkipIfRemote("The -l,--latest option not supported with remote")
+		var names [2]string
+
+		for i := 0; i < 2; i++ {
+			session := podmanTest.Podman([]string{"run", ALPINE, "true"})
+			session.WaitWithDefaultTimeout()
+			Expect(session).Should(Exit(0))
+
+			session = podmanTest.Podman([]string{"container", "inspect", "-l", "--format", "{{.Name}}"})
+			session.WaitWithDefaultTimeout()
+			Expect(session).Should(Exit(0))
+			names[i] = session.OutputToString()
+			Expect(names[i]).To(Not(Equal("")))
+
+			session = podmanTest.Podman([]string{"rm", "-l"})
+			session.WaitWithDefaultTimeout()
+			Expect(session).Should(Exit(0))
+		}
+		Expect(names[0]).To(Not(Equal(names[1])))
```

Comment:
```suggestion
		Expect(names[0]).ToNot(Equal(names[1]), "names are equal, is the RNG seeded correctly?")
```

A description to the actual error will go a long way if this ever is failing.

Also isn't this whole test flaky by default? I mean technically speaking there is nothing wrong with returning the same name twice. Sure it is unlikely, I looked at the code and there are `25487` possible names, so the chance of this picking the same name is 1/25487.

Reply:
There's two names being generated, so I believe the odds are more like 1/25487^2 but my statistics expertise is at least 30 years old :grin: